Sources/Kinect/Microsoft.Psi.AzureKinect.x64/AzureKinectSensorConfiguration.cs

```diff
@@ -11,6 +11,22 @@ namespace Microsoft.Psi.AzureKinect
     /// </summary>
     public class AzureKinectSensorConfiguration
     {
+        /// <summary>
+        /// Enum for powerline frequency.
+        /// </summary>
+        public enum PowerlineFrequencyTypes

```

Comment:
I named it `PowerlineFrequencyTypes` to prevent naming conflict with the variable name `PowerlineFrequency` used in the configuration. I don't know what is the best convention in this case.